.github/actions/k8s-ci/action.yml

```diff
@@ -0,0 +1,70 @@
+name: 'K8S CI'
+description: 'K8s integration tests'
+inputs:
+  kind-config:
+    description: 'KIND configuration'
+    required: true
+
+  test-target:
+    description: 'Test target'
+    required: true
+  test-range:
+    description: 'Test range'
+    required: false
+    default: ''
+  values-file:
+    description: 'Helm values file'
+    required: true
+
+  docker-images-folder:
+    description: 'Docker build cache folder'
+    required: false
+    default: '/tmp/.docker-images'
+runs:
+  using: composite
+  steps:
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
```

Comment:
Why is this action required?

Reply:
It's not needed; left it there while exporting the docker image build elsewhere. I will clean it up.